pressure_converter:
  ros__parameters:
    zero_r:
      - -27455372.92
      - 14459386.46
      - 17952654.76
      - -10605254.74
    zero_l:
      - -14063383.6
      - 30331751.92
      - -2034247.26
      - 48130515.2
    scale_r:
      - 2.879445523352348e-06
      - -2.252145560713656e-05
      - -2.754225471994077e-06
      - 2.763926644524643e-06
    scale_l:
      - 2.879445523352348e-06
      - -2.252145560713656e-05
      - -2.754225471994077e-06
      - 2.763926644524643e-06